release-policies.md

```diff
@@ -1,52 +1,54 @@
 # Release Policies
 
-The .NET team uses the following policies for [.NET releases](releases.md). They define what you can expect when you use a given .NET release.
+The .NET team uses the following policies for [.NET releases](releases.md).
 
 ## Release cadence
 
-New major .NET versions are released annually in November. .NET 5.0 was the first version to be released according to this schedule. Minor releases may also be released, with no pre-defined or regular schedule.
+New major .NET versions are released annually in November, at [.NET Conf](https://www.dotnetconf.net/).
 
-## Release types
+Patch updates are released monthly on the second tuesday of each month, also known as Patch Tuesday.
 
-Each .NET release is defined (prior to initial release) as either **Preview**, **Current**, or **Long Term Support (LTS)**. The difference is support time frame, as defined below:
+Minor versions of the .NET SDK are releases approximately quarterly. These are known as [feature bands](https://docs.microsoft.com/en-us/dotnet/core/releases-and-support#feature-bands-sdk-only).
```

Comment:
Suggestion: `are releases approximately quarterly` sounds strange to me. Maybe `release approximately quartlery` would be better.

Reply:
```suggestion
Minor versions of the .NET SDK are released approximately quarterly. These are known as [feature bands](https://docs.microsoft.com/dotnet/core/releases-and-support#feature-bands-sdk-only).
```